src/include/connection.h

```diff
@@ -281,6 +281,11 @@ struct __wt_connection_impl {
     WT_SPINLOCK optrack_map_spinlock; /* Translation file spinlock. */
     uintmax_t optrack_pid;            /* Cache the process ID. */
 
+#ifdef HAVE_CALL_LOG
+    /* File stream used for writing to the call log. */
+    WT_FSTREAM *call_log_fst;
```

Comment:
it should be of type WT_FH, for reference please check     WT_FH *log_fh;         /* Logging file handle */ in log.h

Reply:
Why is that the case? I thought a WT_FSTREAM would be required to use `__wt_fprintf` to output the json string to the call log?

Reply:
Yes, you can ignore my comment.